src/main/webapp/index.html

```diff
@@ -16,7 +16,31 @@
   </head>
   <body>
     <div id="content">
-
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <span class="navbar-brand mb-0 h1">InterviewMe</span>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
You should split this <button> tag, possibly twice, so that it is not overly long.

Reply:
Done